src/V4Router.sol

```diff
@@ -66,12 +65,11 @@ abstract contract V4Router is IV4Router, BaseActionsRouter, DeltaResolver {
                     recipient := calldataload(add(params.offset, 0x20))
                 }
 
-                int256 delta = poolManager.currencyDelta(address(this), currency);
```

Comment:
i think you can now remove the transient state library from this file

Reply:
and probably from posm too?

Reply:
removed from router, we need in posm for close